### ISSUE TITLE
Add switch to Home app to turn engine on and off

### ISSUE THREAD

**Issue by vgnome** [WonkiDonk/homebridge-jlr-remote#1]
This is re: our email about being able to say "Hey Siri, start the car."

You mentioned this would be possible by adding a switch for the engine into the Home app, and then I could define a scene called "Start the Car" to perform this action.

Thank you!

**Comment by jagpreetbatth** [comment]
Any idea how to download the platform?

**Comment by william-cowell-mf** [comment]
Yeah, this feature is on the roadmap. This plug-in is taking a little longer to develop as I'm using it as an opportunity to teach a student to code. But watch this space...

**Comment by WonkiDonk** [comment]
I'm looking for a volunteer with TypeScript / JavaScript experience with a Jaguar or Land Rover with an internal combustion engine that can help me test this.
